### DISCUSSION TITLE
i dont know how to run it in my mac ughh

### DISCUSSION THREAD

**Discussion by bvazfra** [Dpeta/pesterchum-alt-servers#71] [Q&A] [answered]
help

**Comment by Dpeta** [comment, marked as answer]
EDIT: [Normal releases should just work now :3](https://github.com/Dpeta/pesterchum-alt-servers/releases)

~~Yeah- Mac setup is a bit painful rn, the builds barely work ;v;
Probably best to just run from source, you could do that by doing something along the lines of this:~~

~~1. Open the app 'Terminal'
2. Paste in the command from the top of https://brew.sh/
3. After brew is installed, run brew install python3 (Python might already be installed, in which case this isn't needed)
4. After Python is installed, run ``python3 -m pip install pygame PyQt5 feedparser python-magic ostools requests`` to install Pesterchum's dependencies.
5. Download https://github.com/Dpeta/pesterchum-alt-servers/archive/refs/heads/py3_pyqt5.zip and extract the contents to a folder of choice.
6. Run ``cd [folder]`` in your terminal with [folder] replaced with wherever you put the contents of the zip file.
7. Run python3 pesterchum.py~~

~~It might uh, be easier to just ask on the support server though </3: https://discord.gg/UT2fGAr22q~~

**Reply by bvazfra** [reply]
damn thats confusing

**Reply by Dpeta** [reply]
Yeahh <///3
If you wanna do it this way I'd rly recommend joining the server 4 help if you run into any issues ;w;
I'll probably have an easier method up sometime soon- Mybe builds will work without pygame </3

**Reply by bvazfra** [reply]
okay thank you! :o>

**Reply by Dpeta** [reply]
Updated the answer :3👍
